### PR TITLE
Update delete command

### LINKED ISSUE #3: Delete contact
As a user, I want to delete contacts so that I can remove contacts that I don't need anymore.

This task includes:
1. Creating the delete command
2. Deleting the task
3. Updating the UI
4. Removing it from storage/data file

### PR DESCRIPTION
**Pull request by yanhwee**
Resolves #3

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -84,4 +86,15 @@ public interface Model {
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
+
+    /**
+     * Returns a person that satisfies the predicate.
+     * @return
```
Comment (jialegitgud) [inline src/main/java/seedu/address/model/Model.java:92]: Perhaps removing this unncessary return would be cleaner

File: src/main/java/seedu/address/logic/Messages.java [hunk 2]
```diff
@@ -15,6 +15,7 @@ public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
```
Comment (jialegitgud) [inline src/main/java/seedu/address/logic/Messages.java:17]: Can be removed if not in use

**Review by jialegitgud — COMMENTED** [review]
LGTM, with just minor clean fixes

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (yanhwee) [reply src/main/java/seedu/address/model/Model.java:92]: Okay

File: src/main/java/seedu/address/logic/Messages.java (hunk 2 above)
Comment (yanhwee) [reply src/main/java/seedu/address/logic/Messages.java:17]: Yeap those are in use by general other code

**Pull request merged**